build.rs

```diff
@@ -55,7 +57,10 @@ fn main() -> Result<(), Box<dyn Error>> {
     // Let the code know if it should use atomic-polyfill or not, and what aspects
     // of polyfill it requires
     match &target[..] {
-        "riscv32i-unknown-none-elf" | "riscv32imc-unknown-none-elf" => {
+        "riscv32i-unknown-none-elf"
+        | "riscv32imc-unknown-none-elf"
+        | "avr-atmega328p"
```

Comment:
should be `avr-unknown-gnu-atmega328`

Reply:
If I perform this modification it malfunctions like this:

```
  --> /home/thoth/src/heapless/src/pool/singleton/arc.rs:88:26
   |
88 | use core::sync::atomic::{AtomicUsize, Ordering};
   |                          ^^^^^^^^^^^ no `AtomicUsize` in `sync::atomic`

error[E0432]: unresolved import `core::sync::atomic::AtomicPtr`
  --> /home/thoth/src/heapless/src/pool/llsc.rs:10:26
   |
10 | use core::sync::atomic::{AtomicPtr, Ordering};
   |                          ^^^^^^^^^
   |                          |
   |                          no `AtomicPtr` in `sync::atomic`
   |                          help: a similar name exists in the module: `AtomicI8`

error[E0432]: unresolved import `core::sync::atomic::AtomicUsize`
  --> /home/thoth/src/heapless/src/spsc.rs:92:26
   |
92 | use core::sync::atomic::{AtomicUsize, Ordering};
   |                          ^^^^^^^^^^^ no `AtomicUsize` in `sync::atomic`

error[E0392]: parameter `T` is never used
   --> /home/thoth/src/heapless/src/pool/mod.rs:263:17
    |
263 | pub struct Pool<T> {
    |                 ^ unused parameter
    |
    = help: consider removing `T`, referring to it in a field, or using a marker such as `PhantomData`
    = help: if you intended `T` to be a const parameter, use `const T: usize` instead
```